08-wt-gradebook.Rmd

```diff
@@ -154,7 +174,9 @@ View the new data frame and note which columns were selected for this new data f
 view(classwork_df)
 ```
 
-## Visualize Data
+## Analysis and results
```

Comment:
okay with this heading for the subsequent sections (for which I indexed the headings "down" one level, i.e., level 2 headings become level 3)?